clients/client-shell/cmds/signin/signin.go

```diff
@@ -84,7 +86,7 @@ func cmdSignin(cmd *cobra.Command, _ []string) error {
 			fmt.Fprintln(cmd.OutOrStdout(), "export TASKCLUSTER_ACCESS_TOKEN='"+qs.Get("accessToken")+"'")
 			fmt.Fprintln(cmd.OutOrStdout(), "export TASKCLUSTER_ROOT_URL='"+rootURL+"'")
 		}
-		fmt.Fprintln(cmd.OutOrStderr(), "Credentials output as environment variables")
+		log.Infoln("Credentials output as environment variables")
```

Comment:
I think we'll want the user to see this even without `-v`.  What if we made the default level "Info" and changed most of the `Infoln` to `Debugln` here?

Reply:
This makes sense. This way all the current `fmt.Fprintln` would still be visible and for further logging, we can use `log.Debug`

Reply:
In fact I think most of the stuff that you've changed to use `log` in this PR is at the right level -- it's just this particular line that I suspect users will miss if it's not seen.

Reply:
We can change the default level to `InfoLevel` since that comes in precedence as compared to `DebugLevel`. That way all the current `Infoln` will be visible but further debug messages will need to written with `Debugln` and only visible with `-v` on.